src/components/vm/nics/nicBody.jsx

```diff
@@ -97,7 +97,8 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
             {
                 name: 'direct',
                 desc: 'Direct attachment',
-                detailParagraph: _("Uses the macvtap driver to attach the guest's NIC directly to a specified physical interface of the host machine.")
+                detailParagraph: _("Uses the macvtap driver to attach the guest's NIC directly to a specified physical interface of the host machine."),
+                detailHeadline: _("This config is suitable where high performance or enhanced security is required. Varying degrees of isolation between guests, or between the guest and the host are possible, depending on the mode and the capabilities of the network to which the physical interface is connected.")
```

Comment:
Can you bring the headline above the paragraph as they are presented in the UI? 

Shall we start with 'This is the recommended config' like the two options above? 

Also where did you find this information? (high performance ... enhanced security ...)

Shouldn't we mention in the detailParagraph about the VEPA which is the mode it will use by default if chosen?

Reply:
I'm trying to sum up all the use cases of the four modes in a couple of words. I think adding extra detail it's too helpful since cockpit-machines doesn't let you change the mode.

The background info comes from https://libvirt.org/formatdomain.html#direct-attachment-to-physical-interface from which I take away:

`vepa`: used to offload switching from the host to the network (performance)  
`bridge`: I think this avoids some overhead for VM-to-VM traffic, but VM-to-host communication is not possible (maybe unless you have a VEPA-capable switch, not sure?) (performance)  
`private`: used when you want to explicitly prevent VM-to-VM and VM-to-host traffic (security)
`passthrough`: for SR-IOV (performance)

But since the default mode is `vepa`, I will add some more details about this use case & the requirement of a VEPA-capable switch.

Reply:
I'll move the headline above the paragraph for all the options, in a separate commit.

Reply:
But cockpit-machines will automatically use VEPA mode - why should be summarize the rest?

If then the user goes and adjusts the XML they probably are fully aware of what they are doing.